### ISSUE TITLE
Sometimes the top menu bar captures focus when typing keyboard shortcuts

### ISSUE THREAD

**Issue by burnflare** [CS2103AUG2016-W11-C2/main#50]
It's a UI bug.

I know this is not descriptive. I'll update this take with details soon.

**Comment by burnflare** [comment]
Here it is:
After a type a command, I like to use the ⌥+Left/Right key to move the cursor by whole words left/right. But for some reason, whenever I hold the ⌥ key, the top menu bar grabs focus immediately.

<img width="133" alt="screen shot 2016-10-12 at 10 44 37 am" src="https://cloud.githubusercontent.com/assets/104150/19295694/e3f08344-9068-11e6-8b6a-b98adb7d8aa4.png">

**Comment by fanwgwg** [comment]
Okay I'll try to go and look into the code

**Comment by fanwgwg** [comment]
I found that this is something that menu bar will do by default. `alt` always activates focus of menu bar item, and I couldn't find a way to disable i, or override it. However, I think this can be good, cuz it allows user to navigate menubar through keyboard. After pressing `alt`, pressing `lef/right/up/down_key` allows user to navigate through the menubar, and then press `enter` will activite the item. Also, to make it better, I changed the shortcut of _Help_ to `⌥ H`, so that the user won't get confused.
